Components/Card/card.css

```diff
@@ -0,0 +1,202 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+.container {
+    max-width: var(--width-xxxs);
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin: var(--space-sm);
+    box-shadow: RGBA(0, 0, 0, 0.24) 0px 3px 8px;
+    border-radius: var(--space-xxs);
+}
+
+.content {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.content-row {
+    display: flex;
+    flex-direction: row;
```

Comment:
Here you can remove `flex-direction: row` as its the default value

Reply:
👍🏼